### ISSUE TITLE
Add resistors to usb cc pins

### ISSUE THREAD

**Issue by david-sawatzke** [wuxx/icesugar-nano#1]
The board currently doesn't work when connected to a usb-c port, because the resistors on cc1/cc2 are missing. To fix this resistors need to be added, similar to the ones on nanoDLA (https://raw.githubusercontent.com/wuxx/nanoDLA/master/hardware/v1.3/nanoDLA.pdf).

**Comment by wuxx** [comment]
yes, next version will fix this problem, thanks

**Comment by abderraouf-adjal** [comment]
@wuxx Hi, is this issue fixed in v1.2?
Also, may you upload v1.2 sch and and its related PCB files?

**Comment by VIPQualityPost** [comment]
I don't think this was fixed.. my board (v1.2 shipped 6/23) does not have these resistors either.
Adding some 5.1k makes it work with USB C ports.
![image](https://github.com/wuxx/icesugar-nano/assets/46805787/b204e5b0-33ae-401a-9608-19e347e6b3a5)

**Comment by stavinsky** [comment]
same for me. works only via adapter or with usb A cable
spend about an hour until found this issue. Thanks. Mine is also 1.2.

**Comment by babayoussef** [comment]
The same problem for me, Hope a new version will come out soon.

**Comment by fred-corp** [comment]
Same here... v1.2 in 2025, no 5.1k resistors..
